WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -11965,7 +11973,7 @@
 		};
 		4598298028574688003A9AFE /* Inject */ = {
 			isa = XCSwiftPackageProductDependency;
-			package = 4598297F28574688003A9AFE /* XCRemoteSwiftPackageReference "Inject.git" */;
+			package = 4598297F28574688003A9AFE /* XCRemoteSwiftPackageReference "Inject" */;
```

Comment:
Has Inject changed for any particular reason? What happens if you revert these changes?

Reply:
Good catch! And that's a good question: 

The only difference I'm seeing is that when I revert the changes is that I can see the iOS 15.0 minimum support warnings for YosemiteTests related to `kUTTypeJPEG` deprecation.

This most likely comes from me switching to iOS15 temporarily when working on this, as at the same time we were upgrading the app minimum version for the app and I wanted to see if my changes came with potential deprecations. Then when I switched back to iOS14, I may have missed the changes in `project.pbxproj`.

I see these changes are [part of trunk already](https://raw.githubusercontent.com/woocommerce/woocommerce-ios/trunk/WooCommerce/WooCommerce.xcodeproj/project.pbxproj), but I reverted here: da5e40177aed121e1ef301c86ac531d6025c010d as this change shouldn't go with my PR anyway 👍